### PR TITLE
Add histogram title

### PR DESCRIPTION
**Pull request by akopich**
In accordance with Carsten's proposal, a histogram title
should be added. It should report the function name
in bold and dataset size.

Tests: Manual
Bug: http://b/223156315

### CODE REVIEW

File: src/OrbitQt/orbitlivefunctions.ui [hunk 1]
```diff
@@ -54,6 +54,31 @@
         <string>Histogram</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
+        <item>
+         <widget class="QLabel" name="histogram_title_">
+          <property name="maximumSize">
```
Comment (florian-kuebler) [inline src/OrbitQt/orbitlivefunctions.ui:59]: Is that needed?

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks

File: src/OrbitQt/orbitlivefunctions.ui (hunk 1 above)
Comment (akopich) [reply src/OrbitQt/orbitlivefunctions.ui:59]: Yes, otherwise the label takes half of the vertical space.

Comment (florian-kuebler) [reply src/OrbitQt/orbitlivefunctions.ui:59]: I was more wondering about the width value.

File: src/OrbitQt/HistogramWidget.cpp [hunk 2]
```diff
@@ -320,4 +310,22 @@ HistogramWidget::GetSelectionRange() const {
 
 void HistogramWidget::EmitSignalSelectionRangeChange() const {
   emit SignalSelectionRangeChange(GetSelectionRange());
-}
\ No newline at end of file
+}
+
+constexpr size_t kMaxFunctionNameLengthForTitle = 80;
+
+[[nodiscard]] QString HistogramWidget::Title() const {
```
Comment (beckerhe) [inline src/OrbitQt/HistogramWidget.cpp:317]: As per the style guide, this function should either be called `title` (to imitiate a public member) or `GetTitle`. I think the latter is more appropriate in this case.

File: src/OrbitQt/orbitlivefunctions.ui (hunk 1 above)
Comment (akopich) [reply src/OrbitQt/orbitlivefunctions.ui:59]: It's default. Do you think I should just remove the line?

Comment (akopich) [reply src/OrbitQt/orbitlivefunctions.ui:59]: Yep, just done that. The title has disappeared.

File: src/OrbitQt/HistogramWidget.cpp [hunk 3]
```diff
@@ -207,11 +203,7 @@ void HistogramWidget::paintEvent(QPaintEvent* /*event*/) {
   DrawHorizontalAxis(painter, axes_intersection, histogram, horizontal_axis_length, MinValue());
   DrawVerticalAxis(painter, axes_intersection, vertical_axis_length, max_freq);
 
-  if (IsSelectionActive()) {
-    DrawTitle(painter, absl::StrFormat("Selection over %u points", histogram.data_set_size));
-  } else {
-    DrawTitle(painter, function_data_->name);
-  }
+  emit SignalTitleChange(GetTitle());
```
Comment (beckerhe) [inline src/OrbitQt/HistogramWidget.cpp:206]: This is also in paintEvent and needs to be moved. Sorry, I forgot to make a separate comment.

File: src/OrbitQt/orbitlivefunctions.ui (hunk 1 above)
Comment (florian-kuebler) [reply src/OrbitQt/orbitlivefunctions.ui:59]: Thanks for checking.

**Pull request merged**